src/app/grid/grid-sample-pinning/grid-toolbar-pinning.component.html

```diff
@@ -1,4 +1,4 @@
-<div [ngClass]="{'grid__wrapper': true, 'dark-theme': dark === true }" style='width: 100%;'>
+<div [ngClass]="{'grid__wrapper': true, 'dark-theme': dark === 'true' }" style='width: 100%;'>
```

Comment:
After keeping the `dark` value boolean, you will be able to use a simpler expression here:

```
<div [ngClass]="{'grid__wrapper': true, 'dark-theme': dark }" style='width: 100%;'>
```
Then it seems even nicer to rename the constant to smth more descrptive, for example `useDarkTheme`, and then the template seems even more readable:

```
<div [ngClass]="{'grid__wrapper': true, 'dark-theme': useDarkTheme }" style='width: 100%;'>
```

Reply:
Ok I'm on it